### PR TITLE
Cognitive Services Ver.2.0.0

### PR DESCRIPTION
**Pull request by felixwa**
1. Everything is re-generated with the latest AutoRest (by
generate.cmd).
2. With the latest swagger file change, now Location is a parameter of
the method, not a property of client class, this is a breaking change.
3. Change the UT accordingly.
4. Bump the version to 1.1.0.

## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/CognitiveServices/Management.CognitiveServices/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Cognitive Services Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Cognitive Services management functions for managing Microsoft Azure Cognitive Services accounts.")]
 
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyVersion("1.1.0.0")]
```
Comment (shahabhijeet) [inline src/SDKs/CognitiveServices/Management.CognitiveServices/Properties/AssemblyInfo.cs]: @felixwa this is not needed. We usually bump up assemblyverion on major version update.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/SDKs/CognitiveServices/Management.CognitiveServices/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (felixwa) [inline src/SDKs/CognitiveServices/Management.CognitiveServices/Properties/AssemblyInfo.cs]: @shahabhijeet , you mean both AssemblyVersion and AssemblyFileVersion need not to be changed or only AssemblyVersion need not to be changed? 
Note this PR introduced a breaking change (Location from property to method parameter) so we definitely need re-versioning.

Comment (shahabhijeet) [inline src/SDKs/CognitiveServices/Management.CognitiveServices/Properties/AssemblyInfo.cs]: @felixwa what is the usual practice is, you only update AssemblyVersion when you update the major version.
If you have introduced a breaking change, then simply update
AssemblyFileVersion to 2.0.0.0
AssemblyVersion to 2.0.0.0

**Comment by shahabhijeet** [comment]
@azuresdkci test this please

File: src/SDKs/CognitiveServices/Management.CognitiveServices/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (felixwa) [inline src/SDKs/CognitiveServices/Management.CognitiveServices/Properties/AssemblyInfo.cs]: @shahabhijeet , okay then I have reversioned everything to 2.0.0.0

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**